internal/stores/gcs_test.go

```diff
@@ -0,0 +1,150 @@
+package stores
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/fsouza/fake-gcs-server/fakestorage"
```

Comment:
This fake gcs server actually adds a ton of overhead and dependencies. @radsec @natewalck what are your thoughts on this? I could really simplify the test and reduce our dependencies if I just write my own abstraction and fake (in memory instead of via a http server) for a gcs bucket.

Reply:
I'm good either way. In theory, this only bloats the tests, so 🤷

Reply:
I would say, mock/simplify test code where possible. 

If the code is somewhat hard to test, its probs could be simplified or split apart IMHO.

Reply:
I think it's easy to test, even if we go the low dependency route. But for now I'm going to keep this